source/shared/net/desktop-wallet/install-ledger-app.rst

```diff
@@ -108,6 +108,8 @@ You now have to install a custom certificate on the Ledger to ensure that it tru
 
 #. Enter your PIN.
 
+.. _install-app-windows:
```

Comment:
```suggestion
.. _install-ledger-app-windows:
```
Might be good to add "ledger" to the link, in case we get other Windows apps at some point.